assets/static/css/style.css

```diff
@@ -293,6 +293,17 @@ a:focus {
   font-weight: 500;
 }
 
+/* Roadmap section */
```

Comment:
Graphical remark: the Latest news section and the Roadmap section are both printed on while background, breaking section-changing style.  Can you think of some solution to amend this?  E.g. a third different-pink colour background and alternate those? Or a greyish box here or for docs? Basically looking just to separate them a bit better...

Reply:
I've tried with something between white and sepia, but I'm not sure... Also sepia doesn't look too good for the roadmap due to the white boxes..

<img width="1678" alt="image" src="https://user-images.githubusercontent.com/8863238/94294807-a9447280-ff60-11ea-886d-669cd8ddf43f.png">

Maybe inverting those two color so it kinda creates a gradient effect?

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/8863238/94295030-f294c200-ff60-11ea-8bf0-57874d1bf663.png">

Reply:
:+1: for the gradient effect. Looks nice with the slightly darker footer.

Reply:
_Gradient_ effect applied.